WcaOnRails/config/locales/en.yml

```diff
@@ -392,7 +392,7 @@ en:
     connect_wca_id: "Connect your WCA ID to your account!"
   persons:
     index:
-      name_or_wca_id: "Name or WCA ID"
+      name_parts_or_wca_id: "Name parts or WCA ID"
```

Comment:
I think it actually would be okay to continue to just say "name" here, instead of "name parts".